src/ServicePulse.Host/vue/vite.config.ts

```diff
@@ -67,10 +70,7 @@ export default defineConfig({
       external: ["./js/app.constants.js"],
     },
   },
-  server: {
-    headers: {
-      "Content-Security-Policy": createCSPOverrides(port, defaultUrls),
```

Comment:
Why is this now removed?

Reply:
@johnsimons  I removed it by mistake, I keep it removed only locally because it affects my workflow, but forgot to un-stage it during my last commit